// TASK 1
// Что вы думаете? Выполнится ли .catch? Поясните свой ответ.
//
// 	new Promise(function(resolve, reject) {
// 	setTimeout(() => {
// 		throw new Error("Whoops!");
// 	}, 1000);
// }).catch(alert);

//answer
//нет, не выполнится